### ISSUE TITLE
run_openai_gpt.py issues with Adamw

### ISSUE THREAD

**Issue by rabeehk** [huggingface/transformers#837]
Hi
Adamw in this script has parameters not existing anymore, ...
Thanks for updates in advance.

**Comment by thomwolf** [comment]
Yes, so this should be fixed by (your own :) PR #845 Thanks again!

**Comment by rabeehk** [comment]
thanks :)
Best regards,
Rabeeh

**Comment by prrao87** [comment]
Thanks @rabeehk! #845 with flat learning rate gives a good result of 87.2% on ROCStories. Here are the args (the defaults in the file work well).

```
python run_openai_gpt.py  \
  --model_name openai-gpt  \
  --do_train  \
  --do_eval  \
  --train_dataset "./ROCStories/cloze_test_val__spring2016 - cloze_test_ALL_val.csv"  \
  --eval_dataset "./ROCStories/cloze_test_test__spring2016 - cloze_test_ALL_test.csv"  \
  --train_batch_size 8  \
  --eval_batch_size 16  \
  --num_train_epochs  3
```

**Comment by rabeehk** [comment]
@prrao87  great :)
